README.md

```diff
@@ -1,10 +1,12 @@
-
 # Coffee Tasks
 
 A [Hacktoberfest](https://hacktoberfest.com/) Present 🎁 to React Beginners
 
 ![image](https://user-images.githubusercontent.com/68426990/192261187-84c1029b-686c-4549-a41d-9f23394a98a9.png)
 
+# Created for **HACKTOBERFEST**! Come learn front-end development with us throughout the month of October!
```

Comment:
I think this section can be ignored for now. Cz

- The message is redundant as it was already stated above and below this section
- The section will shout with a big font